### PR TITLE
Document TF platform handler

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# TensorFlow SavedModel Platform Handler \[Experimental\]
+
+This platform handler enables loading and serving of models
+in [TensorFlow SavedModel](https://www.tensorflow.org/guide/saved_model)
+format via python backend. The `model.savedmodel` can be provided within
```
Comment (krishung5) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:33]:
```suggestion
format via Python backend. The `model.savedmodel` can be provided within
```

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+```
+pip install tensorfow==2.13.0
+```
+
+Alternatively, you can create a
+[Python Execution Environment](#using-custom-python-execution-environments)
+with the TensorFlow dependency.
+
+By default, Triton will use the [TensorFlow backend](https://github.com/triton-inference-server/tensorflow_backend)
+to load and serve the saved model. In order to use the python backend with
```
Comment (krishung5) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:51]:
```suggestion
to load and serve the saved model. In order to use the Python backend with
```

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# TensorFlow SavedModel Platform Handler \[Experimental\]
+
+This platform handler enables loading and serving of models
+in [TensorFlow SavedModel](https://www.tensorflow.org/guide/saved_model)
+format via python backend. The `model.savedmodel` can be provided within
+the triton server model repository without `model.py` and backend will
+automatically use a pre-baked platform handler (`model.py`) to load and
```
Comment (krishung5) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:35]: Maybe we can link to the model.py file that is under the same directory?

**Review by krishung5 — COMMENTED** [review]
Just some minor comments, otherwise lgtm!

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# TensorFlow SavedModel Platform Handler \[Experimental\]
```
Comment (Tabrizian) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:29]: Can we not mention "platform handlers" for now? @nnshah1 and I were discussing an idea that "platform handlers" might become just a regular backend in Triton so we may not need a new concept. For now, we can change the title to "Serving Tensorflow SavedModels using Python Backend".

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+This platform handler enables loading and serving of models
+in [TensorFlow SavedModel](https://www.tensorflow.org/guide/saved_model)
+format via python backend. The `model.savedmodel` can be provided within
+the triton server model repository without `model.py` and backend will
```
Comment (Tabrizian) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:34]: Would be good to share a blueprint of repository structure using `tree`.

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# TensorFlow SavedModel Platform Handler \[Experimental\]
+
+This platform handler enables loading and serving of models
```
Comment (Tabrizian) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:31]: "Starting from 23.07 release, you can serve TF saved model files using Python backend.."

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+in [TensorFlow SavedModel](https://www.tensorflow.org/guide/saved_model)
+format via python backend. The `model.savedmodel` can be provided within
+the triton server model repository without `model.py` and backend will
+automatically use a pre-baked platform handler (`model.py`) to load and
+serve the TF SavedModel. The handler can [auto-complete](../../../../README.md#auto_complete_config)
+the missing model configuration.
```
Comment (Tabrizian) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:37]: Can we have quick start guide that mentions how this works end-to-end?

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+```
+backend: "python"
+platform: "tensorflow_savedmodel"
+```
+
+Following are few known limitations of the TF platform handler:
+- GPU execution is not supported.
+- List of requests received in model [`execute`](../../../../README.md#execute) function are
+not run in a single batch but one after the other.
```
Comment (Tabrizian) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:63]: Would be good to have a section on how this compares with serving TF models in Tensorflow backend.

**Review by Tabrizian — CHANGES_REQUESTED** [review]

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md (hunk 6 above)
Comment (Tabrizian) [reply src/resources/platform_handlers/tensorflow_savedmodel/README.md:31]: After discussing this more with @nnshah1, how about adding a note that this is added "Experimentally in 23.07"? Also, we need a line that says "This feature is subject to change and removal and should not be used in production".

Comment (tanmayv25) [reply src/resources/platform_handlers/tensorflow_savedmodel/README.md:31]: Sounds good to me..

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md (hunk 7 above)
Comment (tanmayv25) [reply src/resources/platform_handlers/tensorflow_savedmodel/README.md:37]: I will add the quickstart in a separate PR that will not be cherry-picked to 23.07.

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md (hunk 8 above)
Comment (tanmayv25) [reply src/resources/platform_handlers/tensorflow_savedmodel/README.md:63]: Sure!

**Review by Tabrizian — APPROVED** [review]

File: src/resources/platform_handlers/tensorflow_savedmodel/README.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Serving Tensorflow SavedModels using Python Backend \[Experimental\]
+
+*NOTE*: This feature is subject to change and removal, and should not
+be used in production.
+
+Starting from 23.07, we are adding an experimental support for loading
```
Comment (nnshah1) [inline src/resources/platform_handlers/tensorflow_savedmodel/README.md:34]:
```suggestion
Starting from 23.07, we are adding experimental support for loading
```

**Review by krishung5 — APPROVED** [review]

**Review by nnshah1 — APPROVED** [review]

**Pull request merged**